server_installation/topics/operating-mode/crossdc/proc-setting-up-infinispan.adoc

```diff
@@ -0,0 +1,160 @@
+[id='setting-up-infinispan-{context}']
+= Setting Up {jdgserver_name} {jdgserver_version_latest} Clusters
+You begin this procedure by creating remote {jdgserver_name} clusters that can back up {project_name} data.
+
+.Prerequisites
+
+* Download and install {jdgserver_name} Server {jdgserver_version_latest}.
+
+[NOTE]
+====
+{jdgserver_name} Server {jdgserver_version_latest} requires Java 11.
+====
+
+* Create a user to authenticate client connections from {jdgserver_name}, for example:
++
+[source,bash,options="nowrap",subs=attributes+]
+----
+$ bin/cli.sh user create myuser -p "qwer1234!"
+----
++
+You specify these credentials in the Hot Rod client configuration when you set up {jdgserver_name}.
+
+* Create an SSL keystore and truststore to secure connections between {jdgserver_name} and {project_name}.
+
+For example, you can use keytool as follows:
+
+[source,bash,options="nowrap",subs=attributes+]
+----
+# Create a keystore to provide an SSL identity to your {jdgserver_name} cluster
+keytool -genkey -alias server -keyalg RSA -keystore server.jks -keysize 2048
+
+# Export an SSL certificate from the keystore
+keytool -exportcert -keystore server.jks -alias server -file server.crt
+
+# Import the SSL certificate into a truststore that {jdgserver_name} can use to verify the SSL identity for {jdgserver_name}
+keytool -importcert -keystore truststore.jks -alias server -file server.crt
+rm server.crt
+----
+
+.Procedure
+
+. Open `infinispan.xml` for editing.
++
+By default, {jdgserver_name} Server uses `server/conf/infinispan.xml` for static configuration such as cluster transport and security mechanisms.
++
+. Configure the {jdgserver_name} cluster transport to perform cross-site replication.
+.. Add the RELAY2 protocol to a JGroups stack.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<jgroups>
+   <stack name="xsite" extends="udp"> <1>
+      <relay.RELAY2 site="LON" <2>
+                    max_site_masters="1000"/> <3>
+      <remote-sites default-stack="tcp"> <4>
```

Comment:
The default `tcp` stack uses IP multicast to find nodes which is unavailable most of the time. 
I would suggest to create a new stack with `TCPPING` like this:
```
<stack name="global-cluster" extends="tcp">
    <!-- Remove MPING protocol from the stack and add TCPPING -->
    <TCPPING initial_hosts="<ip1>[<port>],<ip2>[<port>]" stack.combine="REPLACE" stack.position="MPING"/>
</stack>
```
and change:
```suggestion
      <remote-sites default-stack="global-cluster"> <4>
```
Note: `initial_hosts` contains the list of all Infinispan servers, i.e. both sites.